tests/testthat/test-pkg-load.R

```diff
@@ -0,0 +1,10 @@
+test_that(".onLoad: Unset options get initialised on package load with defaults", {
```

Comment:
We should also test that options that _are_ initialized are retained and not overwritten

Reply:
Good catch, Added this case.